Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -7047,11 +7047,17 @@ public void HandleActUIElement(ActUIElement act)
                     case ActUIElement.eElementAction.SetText:
                         try
                         {
-                            ClearText(e);
+                            if (e != null)
```

Comment:
we already have null check for e at start of the function. so if e is null we will not reach this code. why we need this additional check ?